src/Permission/permission.controller.ts

```diff
@@ -0,0 +1,48 @@
+import express, { Request, Response } from 'express';
+import { badRequest, notFound, ok } from '../common/constants';
+import { PermissionAttributes } from './permission.interfaces';
```

Comment:
Add blank line after package.json dependencies and before internal project files.